docs/user-guide/customization.rst

```diff
@@ -4,67 +4,25 @@
 Customization and Global Configuration
 **************************************
 
-The :file:`configrc` file
-=========================
+HERMES & SWxSOC Configuration
+=============================
 
-This package uses a :file:`configrc` configuration file to customize
-certain properties. You can control a number of key features of such as
-where your data will download to. HERMES packages look for this configuration file
-in a platform specific directory, which you can see the path for by running::
-
-  >>> import hermes_core
-  >>> hermes_core.print_config()  # doctest: +SKIP
-
-Using your own :file:`configrc` file
-=====================================
-To maintain your own customizations, you must place your customized :file:`configrc` inside the appropriate configuration folder (which is based off the operating system you are working on). The `AppDirs module <https://github.com/sunpy/sunpy/blob/main/sunpy/extern/appdirs.py>`_  provided by the `sunpy` package is used to figure out where to look for your configuration file. 
-
-.. warning::
-    Do not edit the configrc file directly in the Python package as it will get overwritten  every time you re-install or update the package.
-
-You can copy the file below, customize it, and then place your customized :file:`configrc` file inside your config folder.
-
-If you work in our developer environment you can place your configuration file in this directory:
-
-.. code-block:: bash
-
-  /home/vscode/.config/hermes_core/
-
-If you do not use our developer environment, you can run the following code to see where to place it on your specific machine as well:
-
-.. doctest::
-
-  >>> from hermes_core import util
-  >>> print(util.config._get_user_configdir())
-  /home/vscode/.config/hermes_core
+The `hermes_core` package uses an inherited configuration file from the `~swxsoc.util.config` module.
```

Comment:
Is it possible to link to the swxsoc docs directly? https://hermes-core--122.org.readthedocs.build/en/122/user-guide/customization.html It looks like it doesn't link via this build

Reply:
It looks like the hermes-core readthedocs can link to the swxsoc docs fine, just that the `config` module is not part of the `swxsoc` api docs. I'm going to make a PR there to update the API docs and then this link should work.

Reply:
https://github.com/swxsoc/swxsoc/pull/16